org.flatpak.onlyoffice.json

```diff
@@ -0,0 +1,68 @@
+{
+    "app-id": "org.onlyoffice.desktopeditors",
+    "base": "io.atom.electron.BaseApp",
+    "base-version": "18.08",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "desktopeditors",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--filesystem=xdg-pictures",
+        "--filesystem=xdg-desktop",
+        "--filesystem=xdg-videos",
+        "--filesystem=xdg-documents",
+        "--filesystem=xdg-download",
+        "--device=all"
+    ],
+    "modules": [
+        {
+            "name": "desktopeditors",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -Dm755 apply_extra /app/bin/apply_extra",
+                "install -Dm755 desktopeditors /app/bin/desktopeditors",
+                "cp /usr/bin/ar /app/bin",
+                "ARCH_TRIPLE=$(gcc --print-multiarch) && cp /usr/lib/${ARCH_TRIPLE}/libbfd-*.so /app/lib"
+            ],
+            "sources": [
+                {
+                    "type": "script",
+                    "dest-filename": "apply_extra",
+                    "commands": [
+                        "ar x onlyoffice-desktopeditors_amd64.deb",
+                        "tar xf data.tar.xz",
+                        "mkdir -p export/share/applications",
+                        "sed 's|Icon=.*|Icon=org.onlyoffice.desktopeditors|; s|Exec=.*|Exec=desktopeditors|;' usr/share/applications/desktopeditors.desktop > export/share/applications/org.onlyoffice.desktopeditors.desktop",
+                        "echo 'StartupWMClass=onlyoffice' >> export/share/applications/org.onlyoffice.desktopeditors.desktop",
+                        "for icon_size in 16 24 32 48 64 128 256; do
+                            install -Dm644 opt/onlyoffice/desktopeditors/asc-de-${icon_size}.png export/share/icons/hicolor/${icon_size}x${icon_size}/apps/org.onlyoffice.desktopeditors.png
+                        done",
+                        "rm -r onlyoffice-desktopeditors_amd64.deb control.tar.gz data.tar.xz debian-binary usr"
+                    ]
+                },
+                {
+                    "type": "extra-data",
+                    "filename": "onlyoffice-desktopeditors_amd64.deb",
```

Comment:
Any reason for not building from source code?

Reply:
I can answer that one... the team behind onlyoffice has persistenly neglected to post build instructions, even in violation of the AGPL 3.0

https://github.com/ONLYOFFICE/DesktopEditors/issues/4

It's cool to see this app come to Flathub, but there are some unsettled license issues which upstream has failed to address.

Reply:
We can't do much here unless we have a fix at least for this upstream.

Reply:
@bilelmoussaoui, I think  we'll stack  on v8 building too long